package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@eqworks/chart-system",
   "private": false,
-  "version": "0.9.6",
+  "version": "0.9.6-alpha.1",
```

Comment:
To make an "alpha" (or beta, anything not production), you should increment at least a patch version, since 0.9.6 is greater than 0.9.6-alpha.1.

This should be a 0.9.7-alpha... and once it's stable enough, drop the suffix to graduate as 0.9.7